app/controllers/extracts_controller.rb

```diff
@@ -10,7 +10,11 @@ def update
                                             classification_id: classification_id)
     authorize extract
 
-    extract.update! extract_params
+    Workflow.transaction do
+      SubjectReduction.delete(extract.subject_reduction_ids)
```

Comment:
Let's say we used this for ASM. Then all of the aggregated line markings would be invisible in the UI, for the period of time between getting an update and the reducer re-running. Could we do a reset without having to delete the reductions before calling the worker?